mantle/kola/tests/rhcos/luks.go

```diff
@@ -176,17 +176,17 @@ func b64Encode(s string) string {
 func mustMatch(c cluster.TestCluster, r string, output []byte) {
 	m, err := regexp.Match(r, output)
 	if err != nil {
-		c.Fatal("Failed to match regexp %s: %v", r, err)
+		c.Fatalf("Failed to match regexp %s: %v", r, err)
 	}
 	if !m {
-		c.Fatal("Regexp %s did not match text: %s", r, output)
+		c.Fatalf("Regexp %s did not match text: %s", r, output)
 	}
 }
 
 func mustNotMatch(c cluster.TestCluster, r string, output []byte) {
 	m, err := regexp.Match(r, output)
 	if err != nil {
-		c.Fatal("Failed to match regexp %s: %v", r, err)
+		c.Fatalf("Failed to match regexp %s: %v", r, err)
 	}
 	if m {
 		c.Fatal("Regexp %s matched text: %s", r, output)
```

Comment:
Should this be `c.Fatalf` also?

Reply:
Good catch! I think so. It's weird that `golanci-lint` didn't point out that.

Reply:
I found out the reason. The default value of `max-same-issues` is 3. So, it only shows the first three same issues.